tasks/main.yml

```diff
@@ -37,24 +37,30 @@
   tags:
       - xpack
 
-- include: elasticsearch-templates.yml
+- include: elasticsearch-template-copy.yml
   when: es_templates
   tags:
       - templates
 
 - meta: flush_handlers
 
+- include: elasticsearch-template-insert.yml
```

Comment:
ah yes good spot. We don't have an idempotent test for this - im going to add one now in the standard tests and update the PR.  I originally intended the template insert file to be where we would do any actions after restarting, but i think we can do the restart and then copy the files and do the insert. I'll update and test.